storage/pebble/registers.go

```diff
@@ -61,7 +92,17 @@ func (s *Registers) Store(
 	height uint64,
 	entries flow.RegisterEntries,
 ) error {
-	//  TODO: add value check once LatestHeight is implemented
+	s.lock.Lock()
```

Comment:
I wonder if we can only wrap the line that changes the latest height value and not the whole function, that way we could still allow concurrency for batch updating the db but only wait on updating the lastest height which should be significantly faster than updating db. I would like a second opinion tho cc @peterargue @zhangchiqing

Reply:
I'll do it with the atomic variable suggested here: https://github.com/onflow/flow-go/pull/4714#discussion_r1332030878